vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -14306,7 +14328,7 @@ quantitykind:SpecificActivity
 .
 quantitykind:SpecificEnergy
   a qudt:QuantityKind ;
-  dcterms:description """\\(\\textbf{Specific Energy}\\) is defined as the energy per unit mass. Common metric units are \\(J/kg\\). It is an intensive property. Contrast this with energy, which is an extensive property. There are two main types of specific energy: potential energy and specific kinetic energy. Others are the \\(\\textbf{gray}\\) and \\(\\textbf{sievert}\\), which are measures for the absorption of radiation. 
+  dcterms:description """\\(\\textbf{Specific Energy}\\) is defined as the energy per unit mass. Common metric units are \\(J/kg\\). It is an intensive property. Contrast this with energy, which is an extensive property. There are two main types of specific energy: potential energy and specific kinetic energy. Others are the \\(\\textbf{gray}\\) and \\(\\textbf{sievert}\\), which are measures for the absorption of radiation.
```

Comment:
Please add ^^qudt:LatexString ;
at the end of the string.

Reply:
Please add ^^qudt:LatexString ;
at the end of the string.

Reply:
^^qudt:LatexString exists at the end of the description, but the text was somehow spitted into multiple lines. i have formatted it.